src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -740,6 +739,8 @@ def _create(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key
             "adminUsername": admin_username
         },
     }
+    if api_version == "2017-07-01":
```

Comment:
What happens when there's a new API version released? Will this list have to be updated?
The method `supported_api_version` at `azure.cli.core.profiles` may be useful here?

Or is this really only for *this* specific API version

Reply:
currently, we are in a very tricky stage. we have two total different RP both running. However, we expect to see only one RP remaining after 2 months. 

In the long term, we shouldn't have to specify api-version. Currently, we want to expose the api-version, only supported by one type of RP to preview clients.

For short, it is temporary.